libraries/exoplayer/src/main/java/androidx/media3/exoplayer/audio/AudioCapabilities.java

```diff
@@ -71,7 +72,7 @@ public final class AudioCapabilities {
       new ImmutableMap.Builder<Integer, Integer>()
           .put(C.ENCODING_AC3, 6)
           .put(C.ENCODING_AC4, 6)
-          .put(C.ENCODING_DTS, 6)
+          .put(C.ENCODING_DTS, 10)
```

Comment:
is this per-DTS spec or is it because the pull-request has MimeTypes to map AUDIO_DTS_EXPRESS and AUDIO_DTS_X to ENCODING_DTS?

Reply:
It's because the "pull-request has MimeTypes to map AUDIO_DTS_EXPRESS and AUDIO_DTS_X to ENCODING_DTS". This mapping is necessary for Video tunnel mode + Audio Direct Passthrough Playback. All DTS audio types will be sent to MTK/Realtek platform via AudioTrack with ENCODING_DTS. Thanks.

Reply:
@christosts I have reset the value of C.ENCODING_DTS back to 6 and added C.ENCODING_DTS_UHD_P2=10 . Hope this is more correct. Kindly see 61a6b995104a3e40a9e27d223eceb8e79452ae2d. Thanks.